src/main/java/S01/Main.java

```diff
@@ -0,0 +1,12 @@
+package S01;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        InputOutputSystem inputOutputSystem = new InputOutputSystem();
+        while (inputOutputSystem.inputString());
```

Comment:
종료되었는지 확인만 하고
while문 안에서 로직을 수행하는 것은 어떨까요?!
```suggestion
        while (~~) {
            ~~
        }
```

Reply:
역할을 분리하려고 while문 안에서 로직을 수행하는 것일까요???

Reply:
네 맞아요!
해당 메서드는 판별도 하고, 화면을 뿌려주는 것도 같이 하고 있기 때문에
다른 사람이 봤을 때, 내부적으로 어떻게 구현되어 있는지 확인해야 하는 문제가 있을 것 같네요 ㅎ

Reply:
네 감사합니다!! 수정하겠습니다!